docs/install.md

```diff
@@ -3,35 +3,38 @@
 This documents any required information either during installation or
 post installation to ensure the operator can function correctly.
 
-## STS Clusters
-
-### Post operator installation
+- [STS Clusters](#sts-clusters)
+    - [Post operator installation](#post-operator-installation)
+        - [Use CredentialsRequest created by the operator](#use-credentialsrequest-created-by-the-operator)
+        - [Use predefined secret](#use-predefined-secret)
+            - [Option 1. Using ccoctl](#option-1-using-ccoctl)
+            - [Option 2. Using AWS CLI](#option-2-using-aws-cli)
 
+## STS Clusters
 In an STS Cluster, `CredentialsRequest`s are not automatically provisioned by
 the **cloud-credential-operator** and the manual intervention done by the
 cluster-admin is required. IAM role and policies as well as the credentials secret need to be provisioned manually for the further consumption by the pods.
-`ccoctl` binary can be used to facilitate this task.
 
 Normally, the **aws-load-balancer-operator** relies on the **cloud-credential-operator**
-to provision the secret for the operated controller using `CredentialsRequest`. And so in an STS cluster this
+to provision the secret for the operated controller using `CredentialsRequest`. However in an STS cluster this
 secret needs to be provisioned manually. The **aws-load-balancer-operator** will wait until the required
 secret is created and available before spawning the **aws-load-balancer-controller** pod.
 
-#### Pre-Requisites
+### Post operator installation
```

Comment:
Just add a little intro. Something like this, but tweak to make it correct.
```suggestion
### Post operator installation

There are three ways to manually create the required `CredentialsRequest`s.  You can use the `ccoctl` binary on ____, but not on MacOS.  Use the `ccoctl` binary to to use the `CredentialsRequest` created by the operator, or use a predefined Secret.  Use the AWS CLI to create a predefined Secret if you prefer not to use `ccoctl`, or are running the process on MacOS.
```

Reply:
An intro is added.